contacts-manage-groups.adoc

```diff
@@ -1,95 +1,94 @@
+. Click btn:[New Message]
+. In the *To* field, type the first few words of your list's name.
+. The list appears as one of the suggestions.
+. Select the list.
+. {product-short} populates the  *To* with all the email addresses from that list.
+. Type in the body of the email.
+. Click btn:[Send].
+
+=== Manage Contact Folders
+You can create folders within *Contacts* and use them to store contacts, lists, and subfolders.
+You can also share contact folders with other users.
+They work much in the same way as email folders.
+You can perform actions like creating folders, subfolders, and moving contacts across folders.
+
+==== Create a Contact Folder
+. Click btn:[Contacts] tab.
+. Hover over *Folders* and click image:graphics/plus.svg[plus icon, width=20].
+. In the text box, type in the name of the folder and press kbd:[Enter].
+** Click outside the text-box or image:graphics/close.svg[close icon, width=20] beside the text box to cancel.
+
+==== Add Contacts to a Folder
+. Click btn:[Contacts] tab.
+. Select contacts to add to a folder.
+. In contact details pane, under image:graphics/folder-move.svg[folder move icon, width=20], click btn:[Move Contacts].
+. In *Choose a Folder* modal, choose a folder to move the selected contacts.
+
+==== Rename a Contact Folder
+. Click btn:[Contacts] tab.
+. Right-click a folder to rename.
+. From the context menu, choose *Rename Folder*.
+. Type the new name for the folder in the text box.
+. Press kbd:[Enter] to save.
+
+==== Move a Contact Folder
+. Click btn:[Contacts] tab.
+. Right-click a folder to move.
+. From the context menu, choose *Move Folder*.
+. Select a folder under which to move this folder.
+
+==== Delete a Contact Folder
+. Click btn:[Contacts] tab.
+. Right-click the folder.
+. From the context menu, choose *Delete Folder*.
+. The folder moves under *Trash*.
+
+==== Create a Contact Folder
+. Click btn:[Contacts] tab.
+. Right click a folder.
+. From the context menu, choose *Create subfolder*.
+. In the text box, type in the name of the folder and press kbd:[Enter].
+** Click outside the text-box or image:graphics/close.svg[close icon, width=20] beside the text box to cancel.
+
+==== Share a Contact Folder
+. Click btn:[Contacts] tab.
+. Right-click a folder to share.
+. From the context menu, choose *Share*.
+. Choose appropriate permissions from the *Sharing Permissions* drop-down.
++
+View:: Users can view all contacts under the shared folder but cannot make changes to that folder.
+
+View, edit, add, and remove:: Users have permission to view and edit the contents of a folder, create new subfolders, present items on your behalf, and delete items from the folder.
+
+View, edit, add, remove, and administer:: Users have permission to view and edit the content of a shared folder, create new subfolders, present on your behalf, delete items from the shared folder, and share the folder with others.
+
+. Enter the email address(s) with whom to share the contact folder.
```

Comment:
I know the previous lines 22 and 25 have been removed/modified, but why do we see lots of changes here line 1-93? Are Line breaks CR, LF or CR+LF changed?

Reply:
While removing the error message in the git diff "No newline at end of file", accidentally I changed the line line breaks.  Fixed to back out to the original one.